packages/cozy-client/src/__tests__/store.spec.js

```diff
@@ -178,6 +179,57 @@ describe('Store', () => {
     })
   })
 
+  describe('getCollectionFromState', () => {
+    it('should return null when the store is empty', () => {
+      const spy = jest.spyOn(global.console, 'warn').mockReturnValue(jest.fn())
+      expect(getCollectionFromState(store.getState(), 'io.cozy.todos')).toBe(
+        null
+      )
+      spy.mockRestore()
+    })
+
+    it('should return document list for given doctype', () => {
```

Comment:
should return all documents for the given doctype
```suggestion
    it('should return document list for given doctype', () => {
```